shelley/chain-and-ledger/executable-spec/test/Test/Utils.hs

```diff
@@ -0,0 +1,8 @@
+module Test.Utils
+  ( all'
+  ) where
+
+import Hedgehog ((===), MonadTest)
+
+all' :: (MonadTest m, Show a, Eq a) => (a -> Bool) -> [a] -> m ()
```

Comment:
Maybe this should have a name like `assertAll`, and a comment explaining what it does, and why it is important to use it (basically, because you'll get a diff output).